module_1_task_14/task_10.py

```diff
@@ -0,0 +1,7 @@
+numbers = [1,2,3,4,5]
```

Comment:
Please add spaces for a better readability
```suggestion
numbers = [1, 2, 3, 4, 5]
```